ui/app/src/Utils/Types.elm

```diff
@@ -30,10 +29,6 @@ type alias Label =
     ( String, String )
 
 
-type alias ApiData a =
-    ApiResponse Http.Error a
```

Comment:
I think I originally brought this in from reading the RemoteData blog post. I think the other had a generic type, and then made a specific type alias for dealing with http requests.